engine/src/main/java/org/terasology/rendering/dag/nodes/ChunksRefractiveReflectiveNode.java

```diff
@@ -77,7 +82,7 @@ public void process() {
      * accommodate the rendering of the water surface from an underwater point of view.
      */
     private void preRenderSetupSceneReflectiveRefractive() {
-        sceneReflectiveRefractive = frameBuffersManager.getFBO("sceneReflectiveRefractive");
+        sceneReflectiveRefractive = dynamicFBM.getFBO(REFRACTIVE_REFLECTIVE_URN);
```

Comment:
In addition to that, when I think about what would the method FBM.get(ResourceUrn) mean something rather than getFBO, I don't assume any other different functionality.  Therefore, agreed!